tests/integration/targets/aci_l3out_extepg/tasks/main.yml

```diff
@@ -18,7 +18,7 @@
       use_ssl: '{{ aci_use_ssl | default(true) }}'
       use_proxy: '{{ aci_use_proxy | default(true) }}'
       output_level: '{{ aci_output_level | default("info") }}'
-    cisco.aci.aci_tenant: ansible_test
```

Comment:
Why remove the cisco.aci here?
Also please add tests to test the actual recursiveness of the function and make sure the whole function is executed in coverage.

Reply:
I think this was a issue introduced by me actually on the FQCN for all tasks where this is a variable called aci_tenant instead of task